src/run.ts

```diff
@@ -84,14 +85,23 @@ function updateEnvFiles() {
 
 // run_fe_locally runs the frontend and its dependencies locally
 // @ts-ignore
-async function run_fe_locally(runner: LabeledProcessRunner, options: { stage: string }) {
-  const stage = options.stage;
-  await writeUiEnvFile(stage, true);
+async function run_fe_locally(runner: LabeledProcessRunner) {
+  const apiUrl = await getCloudFormationStackOutputValue(
+    "seds-jon-cdk",
+    "ApiUrl"
+  );
```

Comment:
Wouldn't this mean that the local app is calling to the api of `seds-jon-cdk`?
I could be confused!

Reply:
Ah ok. Can we change the automatically named local stack's name to be `local` or something?
Something like this:
```ts
  const deployCmd = [
    "cdklocal",
    "deploy",
    "--context",
    "stage=local",
    "--all",
    "--no-rollback",
  ];
```

Reply:
Then that would make more sense to me when we look for the output from the stack